packages/flutter/lib/src/widgets/selection_container.dart

```diff
@@ -179,7 +179,21 @@ class _SelectionContainerState extends State<SelectionContainer> with Selectable
   @override
   SelectedContent? getSelectedContent() {
     assert(!widget._disabled);
-    return widget.delegate!.getSelectedContent();
+    final SelectedContent? content = widget.delegate!.getSelectedContent();
+    if (content == null) return null;
+
+    final String contentStr = content!.plainText;
```

Comment:
Is the `!` needed here? We already assert on the nullability of the local variable on line 183.